.gitignore

```diff
@@ -15,8 +15,8 @@
 .ensime_cache/
 .ensime_lucene
 .generated-mima*
-# The star is required for further !.idea/ to work, see https://git-scm.com/docs/gitignore
-.idea/*
```

Comment:
Actually, the star is not required.  with the current gitignore of this PR, the change of `.idea/vcs.xml` is not ignored and the links on IDEA works.

Reply:
It's because that `.idea/vcs.xml` already tracked by git. 

After removing `*`, when you add some rules like `!.idea/icon.png`, it will not pickup by git.

Reply:
Well we can always add the new file with `-f` option. Git has hint for this:
```
$ git add .idea/foo
The following paths are ignored by one of your .gitignore files:
.idea
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"

$ git add -f .idea/foo
$ git st
On branch improveIgnore
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	new file:   .idea/foo
```

Reply:
`-f` make sense to me.